discord/ext/pages/pagination.py

```diff
@@ -757,6 +757,7 @@ async def send(
         reference: Optional[Union[discord.Message, discord.MessageReference, discord.PartialMessage]] = None,
         allowed_mentions: Optional[discord.AllowedMentions] = None,
         mention_author: bool = None,
+        delete_after: float = None,
```

Comment:
```suggestion
        delete_after: Optional[float] = None,
```

Reply:
mention_author doesn't have Optional[bool] either, I was using that as an example.